addon/components/basic-dropdown-content.ts

```diff
@@ -116,27 +112,29 @@ export default class BasicDropdownContent extends Component<Args> {
 
   @action
   animateIn(dropdownElement: Element): void {
-    if (!this.animationEnabled) return;
     waitForAnimations(dropdownElement, () => {
       this.animationClass = this.transitionedInClass;
     });
   }
 
   @action
   animateOut(dropdownElement: Element): void {
-    if (!this.animationEnabled) return;
     let parentElement = dropdownElement.parentElement;
     if (parentElement === null) return;
     if (this.args.renderInPlace) {
-      parentElement = parentElement.parentElement
+      parentElement = parentElement.parentElement;
+
+      if (isTesting() && parentElement === null) {
```

Comment:
Not sure exactly why but will-destroy modifier (https://github.com/cibernox/ember-basic-dropdown/blob/master/addon/templates/components/basic-dropdown-content.hbs#L30) is getting called in the template after the parent node has been removed from the DOM. Looks like it's only related to Test env.
Related Issues:
emberjs/ember.js#18795